manager/pom.xml

```diff
@@ -47,6 +47,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-reactive-mysql-client</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-reactive-pg-client</artifactId>
+        </dependency>
```

Comment:
We'll need this when running db-syncer with Postgres during tests. Adding in an additional commit.

Reply:
We actually have a bit of a problem with multiple DB vendors atm. The DatabaseHandler injects a vertx `Pool` instance, however the implementation is tied to the `quarkus.datasource.db-kind` property which can only be configured at build time.

Eventually we will want different build profiles and/or modules per DB vendor, but this also complicates testing.

@wburns Any thoughts on how best to overcome this?

Reply:
This really needs to discussed outside of the context of this PR. We should just remove this change from here.

There are 2 options though I can think of. We have to have a different build per DB, or we can define multiple named providers and based on the url the user specifies at runtime we can pick out the appropriate one. The former will produce leaner and simpler code, the latter provides more flexibility.

Reply:
I removed the dependency for now.

Reply:
> This really needs to discussed outside of the context of this PR. We should just remove this change from here.

+1, I just wanted to make sure everyone was aware of the issue.

I have created https://github.com/gingersnap-project/cache-manager/issues/28 to continue the discussion and track the work.